homeassistant/components/smartthings/cover.py

```diff
@@ -90,6 +88,13 @@ def __init__(self, device):
         ):
             self._attr_supported_features |= CoverEntityFeature.SET_POSITION
 
+        if Capability.door_control in device.capabilities:
+            self._attr_device_class = CoverDeviceClass.DOOR
+        elif Capability.window_shade in device.capabilities:
+            self._attr_device_class = CoverDeviceClass.SHADE
+        elif Capability.garage_door_control in device.capabilities:
+            self._attr_device_class = CoverDeviceClass.GARAGE
```

Comment:
Looks like this should be fine since its already used to build `_attr_supported_features`